### ISSUE TITLE
QAMP Checkpoint #2

### ISSUE THREAD

**Issue by IceKhan13** [qiskit-community/ecosystem#84]
### What is the expected behavior?

- [x] Create short description of progress made between checkpoint 1 and November. 
- Upload pictures on project progress
  - [x] Abby's POC for qiskit.org ecosystem page
  - [x] Screenshots of github actions workflows
  - [x] Workflow diagram

**Comment by IceKhan13** [comment]
https://github.com/Qiskit/qiskit.org/pull/2258

<img width="1658" alt="138470673-08da5c55-2d3e-4fe6-a3aa-3f131abab88a" src="https://user-images.githubusercontent.com/8796709/140528974-877124ac-df7d-444e-98c1-25568ff5e7c0.png">
<img width="1652" alt="138470682-9a495d40-d3b8-4ae3-857f-3b40a7a5140e" src="https://user-images.githubusercontent.com/8796709/140528984-6e0cc4d4-921d-41b3-bef7-50a1e9479d81.png">

**Comment by IceKhan13** [comment]
@mickahell @bala-ceg FYI

**Comment by mickahell** [comment]
https://drive.google.com/drive/folders/1yVIkJsdlOlhj20PiCGyFAykQLjSFH9OP

**Comment by mickahell** [comment]
![functional-structure](https://user-images.githubusercontent.com/20951376/140535790-145aaa67-49ca-4dbc-b44b-6bc1001f0632.png)
![sub-wf](https://user-images.githubusercontent.com/20951376/140535792-3242581f-771c-4c5e-9eac-dd41260d2790.png)
![weekly-wf](https://user-images.githubusercontent.com/20951376/140535793-6bfa45ba-91cb-4e1a-8756-cf4df7c33b0b.png)

**Comment by IceKhan13** [comment]
Workflow screenshot

<img width="871" alt="Screen Shot 2021-11-05 at 3 07 40 PM" src="https://user-images.githubusercontent.com/8796709/140565236-bde0646f-19e6-494f-b11b-b6922f2f2479.png">

**Comment by IceKhan13** [comment]
Screenshot of board
<img width="1496" alt="Screen Shot 2021-11-05 at 3 08 58 PM" src="https://user-images.githubusercontent.com/8796709/140565395-017277ab-0f53-4ea9-ba91-26f7e303f547.png">

**Comment by IceKhan13** [comment]
Main achievements:
- submission workflow
- weekly checks
- qiskit.org ecosystem page
- 2 community projects

TODOs:
- more projects
- documentation
- final regression tests
- (optional) configuration in DB
- (optional) badging

**Comment by mickahell** [comment]
Speech:

Since checkpoint 1, we focus our work on the creation of the 2 workflows, the submission and weekly checks workflows. Actually we finished and test them. They work pretty well and we are integrating our first projects.
Also, we prepared a project template with all the files need to configure the perfect project and the workflows checks who need to be pass to be add in the Ecosystem. It gonna be add to GitHub template project system.
Finally, we add some more stuffs, like log level, log color and language support since last checkpoint.

As see with Abby Mitchell, she is making up a view for adding the result of our project on the qiskit.org page as a new tab, Ecosystem. This will present the state of the members of the Ecosystem with their version compatible of Terra, current info, ... and the render on the page gonna be totally random.

We started to work on the badging system, to have dynamic badges who change as the weekly status results stay on success or became failed. It's currently on testing phase. Also, the registration of the configuration project in our DB.
Both of these features are optional for this QAMP.

To finished, we need final no-regression tests and improving the documentation.

**Comment by mickahell** [comment]
Nb of words : 211 
The limit is 500 so we have still lot of space to add more info.

**Comment by IceKhan13** [comment]
```
Since first checkpoint our focus was on finishing 2 main workflows: 
- submission - workflows that triggers after new project candidate has been submitted, it checks tests, style and coverage for project and notify if any issues were found. 
- projects scheduled check - workflow that triggers by cron to make sure all of projects within ecosystem are staying up to date with new versions of Qiskit
Major PRs with workflows were merged and we are running some regression tests at this moment. 

Ecosystem page POC for qiskit.org is also in the works as separate project.

More:
- We also implemented ecosystem template for python based projects as separate repository for new developers to use as starting point.
- Language version support has been added to projects - it will allow to specify versions of language project supports
- 2 projects has been added to ecosystem

Remaining works in our TODO list without specific order:
- badging system (in progress) - dynamic badges that shows status of ecosystem membership and tests status 
- more projects - add more (up to 5) new projects to community tier of ecosystem
- documentation (in progress) - well document all modules, classes and methods for ecosystem
- final regression tests - final manual/regression tests to discover potential bugs and flaws of workflows
- (optional) configuration in DB  - add configuration options for each project into DB
```

@mickahell @bala-ceg

**Comment by mickahell** [comment]
Looks good to me, I didn't dare to make a list of things. But I think it's the best format :)

**Comment by mickahell** [comment]
See https://github.com/qiskit-advocate/qamp-fall-21/issues/6

**Comment by mickahell** [comment]
Closes #84
